xdg-terminal-exec

```diff
@@ -30,12 +34,7 @@ EXECARG=''
 
 OIFS="$IFS"
 
-# default debug (and drop non-numeric value)
-if [ -z "$DEBUG" ] || [ "$DEBUG" != "${DEBUG:##*[!0-9]*}" ]; then
```

Comment:
I wanted to shield -eq from non-numeric values

Reply:
I'm not convinced a debug variable should have such shielding? Someone using it should know better. But if you want, i will add it back. Or it could be changed back to `[ "${DEBUG-0}" = '1' ]`?

Reply:
let's go with `[ "${DEBUG-0}" = '1' ]`

Reply:
Done